### ISSUE TITLE
Upgrade crutches to support elixir 1.2.2

### ISSUE THREAD

**Issue by paolodm** [mykewould/crutches#105]
1.1.0 is the latest version crutches supports. Can we upgrade to the latest and greatest?

**Comment by mykewould** [comment]
I'll work on getting this out this week.

**Comment by duijf** [comment]
Did some work on this today. Tests are green, but we use some functions that are deprecated in 1.2.

**Comment by paolodm** [comment]
@duijf Do you have a pull request I can peruse?

**Comment by duijf** [comment]
@paolodm I added the commits to master (179e0db to d4cc1593, see [here](https://github.com/mykewould/crutches/commits/master)), so currently you can use the following in your `mix.exs` until a new version is pushed:

``` elixir
defp deps do
  # Haven't tested if :ref works with :github instead of :git, you might also need the full hash
  [{:crutches, github: "mykewould/crutches", ref: "d4cc1593"}]
end
```

We still need to stop using the deprecated `Map.get_path` and `Map.fetch_path` in favour of `Kernel.get_in` before we can say we're 1.2 ready. (I don't have a lot of time left this week, so feel free to help :smile:) --- you can find quite a list of locations we use them when you run the tests.

**Comment by paolodm** [comment]
@duijf, removing get_path and fetch_path from crutches didn't seem to be a problem. See [this pr](https://github.com/mykewould/crutches/pull/106/files).

So I'm not sure what we need to change, at least in this repository.

**Comment by CyrusNuevoDia** [comment]
Should the `mix.exs` file specify Elixir `~> 1.2` or `> 1.0`. Our TravisCI builds are failing because of this.

**Comment by paolodm** [comment]
Yes, you are right. Changed back to 1.0 and everything is green. :+1:

**Comment by paolodm** [comment]
This landed as 87b494ea86537173544db2163dc2996a20edc534. Closing.
